### PR TITLE
[wix-style-react] - <Modal/> - update storybook examples

### PR DESCRIPTION
**Pull request by egidijusja**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Review by egidijusja — COMMENTED** [review]
@milkyfruit I've updated existing storybook examples (where possible) with newly introduced prop "screen". Can you suggest if we need new examples for `AnouncementModalLayout` and `MessageModalLayout` as they don't have examples for opening layout with `Modal` component.

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6617

**Comment by milkyfruit** [comment]
@egidijusja example for `MessageModalLayout`:

### Opening a modal
Use `<Modal/>` component to reveal the message modal layout.

```
import React from 'react';

class SimpleExample extends React.Component {
  state = {
    isModalOpened: false,
  };

  openModal = () => this.setState({ isModalOpened: true });

  closeModal = () => this.setState({ isModalOpened: false });

  render() {
    const { isModalOpened } = this.state;
    return (
      <Box>
        <Button onClick={this.openModal}>Open Message Modal</Button>
        <Modal
          isOpen={isModalOpened}
          onRequestClose={this.closeModal}
          screen="desktop"
        >
          <MessageModalLayout
            primaryButtonText="Discard"
            secondaryButtonText="Cancel"
            title="Discard changes"
            onCloseButtonClick={this.closeModal}
            primaryButtonOnClick={this.closeModal}
            secondaryButtonOnClick={this.closeModal}
          >
            <Text>
              Are you sure you want to leave this page without saving the
              changes?
            </Text>
          </MessageModalLayout>
        </Modal>
      </Box>
    );
  }
}
```

**Comment by milkyfruit** [comment]
@egidijusja here's the example for `AnnouncementModalLayout`

### Opening a modal
Use <Modal/> component to reveal the announcement modal layout.
```
import React from 'react';

class SimpleExample extends React.Component {
  state = {
    isModalOpened: false,
  };

  openModal = () => this.setState({ isModalOpened: true });

  closeModal = () => this.setState({ isModalOpened: false });

  render() {
    const { isModalOpened } = this.state;
    return (
      <Box>
        <Button onClick={this.openModal}>Open Announcement Modal</Button>
        <Modal
          isOpen={isModalOpened}
          onRequestClose={this.closeModal}
          screen="desktop"
        >
          <AnnouncementModalLayout
            theme="premium"
            illustration={'generic_upgrade.svg'}
            title="Start Accepting Online Payments"
            primaryButtonText="Upgrade"
            linkText="Learn more"
            onCloseButtonClick={this.closeModal}
            primaryButtonOnClick={this.closeModal}
            linkOnClick={this.closeModal}
          >
            <Text>
              Upgrade your site with a business and ecommerce premium plan to
              start accepting payments.
            </Text>
          </AnnouncementModalLayout>
        </Modal>
      </Box>
    );
  }
}
```

File: packages/wix-style-react/src/AnnouncementModalLayout/docs/examples/SimpleExample.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/* eslint-disable */
+import React from 'react';
+
+class SimpleExample extends React.Component {
```
Comment (mykas) [inline packages/wix-style-react/src/AnnouncementModalLayout/docs/examples/SimpleExample.js:4]: Lets do a functional based example. We are trying to move away from class based examples

Comment (egidijusja) [reply packages/wix-style-react/src/AnnouncementModalLayout/docs/examples/SimpleExample.js]: Good point, changed both newly added examples to functional style.

**Review by mykas — APPROVED** [review]

**Comment by mykas** [comment]
Good job 🚀 Lets merge this and update the changelog

**Pull request merged**